### PR TITLE
NumFOCUS onboarding GitHub requirements

### PR DESCRIPTION
**Pull request by Saransh-cpp**
# Description

> If your project has no GOVERNANCE.md at the root, then create a stub GOVERNANCE.md and include a link to your governance documentation or the appropriate parties to contact for more information.

Should we move the governance model to `GOVERNANCE.md`?

Rendered README here - https://github.com/Saransh-cpp/PyBaMM/tree/numfocus#readme.

XRef #2611

## Type of change

Please add a line in the relevant section of [CHANGELOG.md](https://github.com/pybamm-team/PyBaMM/blob/develop/CHANGELOG.md) to document the change (include PR #) - note reverse order of PR #s. If necessary, also add to the list of breaking changes.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (back-end change that speeds up the code)
- [ ] Bug fix (non-breaking change which fixes an issue)

# Key checklist:

- [ ] No style issues: `$ flake8`
- [ ] All tests pass: `$ python run-tests.py --unit`
- [ ] The documentation builds: `$ cd docs` and then `$ make clean; make html`

You can run all three at once, using `$ python run-tests.py --quick`.

## Further checks:

- [ ] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added that prove fix is effective or that feature works

### CODE REVIEW

**Comment by tinosulzer** [comment]
Thanks!

> Should we move the governance model to GOVERNANCE.md?

Yes

File: README.md
Comment (brosaplanella) [inline README.md:14]: Should we move it to first position?

Comment (Saransh-cpp) [reply README.md]: Looks nice in the first position!

Comment (brosaplanella) [reply README.md]: Being superpicky on the badges, is there a way to have the contributors badge in the same row as the others without breaking the bot workflow? (I am worried that the bot will crash if we remove the empty lines)

Comment (Saransh-cpp) [reply README.md]: Yes, moving the contributors badge to the line with all the badges will make the bot ignore it. But, we can move the badge down to the [contributors](https://github.com/pybamm-team/PyBaMM#-contributors) section. We do have PyPI and conda related badges in the [installation](https://github.com/pybamm-team/PyBaMM#-installing-pybamm) section of the README.

Also, maybe we should have the "downloads" badge on the top?

Comment (brosaplanella) [reply README.md]: I quite like the contributors badge at top, but happy either way. It would be great to have some sort of "downloads metric" badge on top, I can't think of a good way around (two would be too much, and if it's just one it is arbitrary which one we choose).

Comment (Saransh-cpp) [reply README.md]: Yes, I guess the current configuration looks good for now. In the future, if the badge count increases, we can switch to a [table-like structure](https://github.com/sktime/sktime#welcome-to-sktime) - 

image

**Comment by tinosulzer** [comment]
Can this be merged?

**Comment by brosaplanella** [comment]
> Can this be merged?

I think so. Checking #2611 we need to add the campaign link to the website (but NumFOCUS needs to send it to us)and the donation link (once we have GitHub approval). But we can probably go ahead merging this so the badge and info are in main too when we make the release tomorrow.

**Review by tinosulzer — APPROVED** [review]

**Pull request merged**
